src/screens/RegisterEnsSheet.js

```diff
@@ -147,20 +173,20 @@ export default function RegisterEnsSheet() {
             </Inline>
           )}
           <SheetActionButtonRow>
-            {isSuccess && debouncedSearchQuery.length > 2 && (
+            {isSuccess && !isNameInvalid ? (
```

Comment:
up to you but i wonder if it makes sense for `isSuccess` to require `!isNameInvalid`

Reply:
good suggestion!!